packages/jest-mock/src/index.ts

```diff
@@ -5,6 +5,8 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+/// <reference lib="ESNext.Disposable" />
```

Comment:
can we just put this in the closest `tsconfig.json`?

Reply:
Moved it from the top-level `tsconfig.json` into the closest one, but the build requires it both there and here - removing it from either of those two places causes the build to fail.